blobstore-fs/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "blobstore-fs"
+version = "0.1.0"
+edition = "2021"
+resolver = "2"
+
+[dependencies]
+async-trait = "0.1"
+log = "0.4"
+# wasmcloud-interface-blobstore = "0.2"
+wasmcloud-interface-blobstore = { path = "../../interfaces/blobstore/rust"}
```

Comment:
Obviously prior to publish the changes to the interface need to be published so we can change this back to a crates reference.

Reply:
Yes, absolutely.